mypy/typeanal.py

```diff
@@ -1427,6 +1452,16 @@ def analyze_callable_args(
                 and self.refers_to_full_names(arg, ("typing_extensions.Unpack", "typing.Unpack"))
                 or isinstance(arg, UnpackType)
             ):
+                if self.defining_alias and self.has_type_params:
+                    tvar_likes = self.find_type_var_likes(arg)
+                    for name, tvar_expr in tvar_likes:
+                        if (name, tvar_expr) not in self.allowed_alias_tvars:
+                            self.fail(
+                                f'Type variable "{name}" is not included in type_params',
+                                arglist,
+                                code=codes.VALID_TYPE,
+                            )
+                            return None
```

Comment:
Added this to handle `Callable[[Unpack[Ts]], Ret]` when `Ts` is not declared in the type parameters. It was causing a crash.

Reply:
I think I'll revert this part, I don't think it is the correct solution here.

Reply:
OK so looks like this already crashes on master with old aliases https://mypy-play.net/?mypy=master&python=3.12&gist=744b559e656f6f35cd5762bddf979038. I'll open an issue to track it.